src/test/java/com/slack/kaldb/logstore/LuceneIndexStoreImplTest.java

```diff
@@ -359,8 +360,8 @@ private void testS3Snapshot(String bucket, String prefix) throws Exception {
       assertThat(getCount(COMMITS_COUNTER, strictLogStore.metricsRegistry)).isEqualTo(1);
 
       Path dirPath = logStore.getDirectory().toAbsolutePath();
-      Collection<String> activeFiles = logStore.activeFiles();
-
+      IndexCommit indexCommit = logStore.getIndexCommit();
```

Comment:
Let's inline this variable in both locations.

Reply:
sorry can you elaborate on what you mean here?